moler/device/device.py

```diff
@@ -133,6 +155,8 @@ def _clear(cls):
             del device
         cls._devices = {}
         cls._devices_params = {}
+        cls._alias_names = {}  # key is alias, value is real name
```

Comment:
Let's change naming. If you analyse code of `_register_name_for_device()` then we can realize it is searching for first generated-name that has not been used by any device yet.
So, self-explaining attribute should be named:
```
cls._unused_names = {}  # key is public_name, value is internal name (generated from public_name) not used yet by any device
```

Reply:
Done